src/main/webapp/app/exam/participate/summary/exam-participation-summary.component.ts

```diff
@@ -119,6 +119,7 @@ export class ExamParticipationSummaryComponent implements OnInit {
         this.setExamWithOnlyIdAndStudentReviewPeriod();
     }
 
+    // TODO: why do we call this at all? seems we do this for every student at the end of the exam. If at all, we should only load this when results have been published
```

Comment:
Plagiarism cases can get published separately and results must not be published yet on Artemis (e.g. as an Instructor you might want to first handle all plagiarism cases before publishing the results for everyone).

We should not invoke this directly after submitting since obviously there won't be any plagiarism cases yet, but imo it makes sense to load them when manually navigating to the summary page at a later point.